utils/parse.js

```diff
@@ -22,6 +22,10 @@ exports.default = function parse(path, content, context) {
   // always attach comments
   parserOptions.attachComment = true
 
+  // provide the `filePath` like eslint itself does, in `parserOptions`
+  // https://github.com/eslint/eslint/blob/3ec436ee/lib/linter.js#L637
+  parserOptions.filePath = path
```

Comment:
Sorry, not thinking -- relative would be fine as long as it's all against cwd. So maybe path.normalize would be fine